### PR TITLE
feat: homepage

### PR DESCRIPTION
**Pull request by ankit9015**
Homepage Layout as per figma sheet

https://user-images.githubusercontent.com/67504902/154864845-1dfc7b96-1547-456c-84db-74b429d75b32.mp4

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    padding: 1rem 4rem;
+    display: fixed;
+    top: 0;
+}
+
+
+.search-box {
+    justify-self: center;
+    align-self: center;
+    background-color: white;
```
Comment (sushil-mod) [inline main.css]: use color variable

**Review by sushil-mod — APPROVED** [review]
just one comment , good to go

File: main.css [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    width: 40%;
+    height: auto;
+    object-fit: contain;
+}
+
+
+  /*-- ------------------------Product announcement ribbon ends -- */
+
+
+
```
Comment (dev-enforced) [inline main.css]: Can remove these extra spaces. Format files using prettier vscode extension

File: index.html [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+                    <a href="/" class="nav-icon-link">
+                      <span class="badge nav-icon-badge">
+                        <i class="fas fa-user"></i>
+                      </span>
+                      <span class="nav-icon-text">Login</span>
+                    </a>
+                    <a href="/"
+                      class="nav-icon-link">
+                      <span class="badge nav-icon-badge">
+                        <i class="fas fa-heart"></i>
```
Comment (dev-enforced) [inline index.html:50]: Can provide the aria attributes here too as you did with the above icons

**Review by dev-enforced — APPROVED** [review]
The code looks good to me !!Apply the suggestions given by @sushil-mod and then you are good to merge

**Pull request merged**